src/Breadcrumb.tsx

```diff
@@ -30,31 +32,39 @@ const Wrapper = styled.li`
   }
 `
 
-const BreadcrumbBase = ({className, children, theme, ...rest}) => {
+const BreadcrumbBase = styled.nav<SystemFlexProps & SystemCommonProps & SxProp>`
+  display: flex;
+  justify-content: space-between;
+  ${COMMON};
+  ${FLEX};
+  ${sx};
+`
+
+export type BreadcrumbProps = ComponentProps<typeof BreadcrumbBase>
+
+function Breadcrumb({className, children, theme, ...rest}: React.PropsWithChildren<BreadcrumbProps>) {
   const classes = classnames(className, 'Breadcrumb')
   const wrappedChildren = React.Children.map(children, child => <Wrapper theme={theme}>{child}</Wrapper>)
   return (
-    <nav className={classes} aria-label="breadcrumb" {...rest}>
+    <BreadcrumbBase className={classes} aria-label="breadcrumb" theme={theme} {...rest}>
       <Box as="ol" my={0} pl={0}>
         {wrappedChildren}
       </Box>
-    </nav>
+    </BreadcrumbBase>
   )
 }
 
-const Breadcrumb = styled(BreadcrumbBase)`
-  display: flex;
-  justify-content: space-between;
-  ${COMMON};
-  ${FLEX};
-  ${sx};
-`
+type StyledBreadcrumbItemProps = {
```

Comment:
`BreadcrumbItemProps` is defined on line 107. `StyledBreadcrumbItemProps` is an internal type that contains all the prop types minus the props that come from styled-components. We prefix name with `Styled*` to avoid naming conflicts.